### PR TITLE
Fix Connection Details View, Remove Plugin Project Command

### PR DESCRIPTION
**Pull request by JoshSmithXRM**

### CODE REVIEW

**Comment by JoshSmithXRM** [comment]
This PR should resolve #4 & #7

**Comment by Power-Maverick** [comment]
I do not want to remove the command from the `package.json` as it also has references to other places in the code. Please remove the changes to `package.json`. I may have a way to just hide it. 
Changes to `ConnectionDetailsView` looks good. Thanks for catching that bug and resolving it as well.

File: package.json [hunk 1]
```diff
@@ -51,11 +51,6 @@
         },
         "menus": {
             "explorer/context": [
-                {
```
Comment (Power-Maverick) [inline package.json:54]: Remove this please. Thanks

File: package.json [hunk 2]
```diff
@@ -208,12 +203,6 @@
                 "when": "false",
                 "category": "%dvdt.explorer.category%"
             },
-            {
```
Comment (Power-Maverick) [inline package.json:211]: Remove this please. Thanks

**Review by Power-Maverick — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (JoshSmithXRM) [reply package.json]: I have reverted this change.

File: package.json (hunk 1 above)
Comment (JoshSmithXRM) [reply package.json]: I have reverted this change.

**Comment by JoshSmithXRM** [comment]
> I do not want to remove the command from the `package.json` as it also has references to other places in the code. Please remove the changes to `package.json`. I may have a way to just hide it.
> Changes to `ConnectionDetailsView` looks good. Thanks for catching that bug and resolving it as well.

Understood. I love what you have done with the project so far. I am definitely excited to see the plugin package feature added in. I've worked with PAC CLI and it definitely makes generating that package for the first time very simple. Some support for CrmSvcUtility for early bound classes would also be a welcomed addition in my opinion.

**Review by Power-Maverick — APPROVED** [review]

**Pull request merged**